src/test/java/seedu/duke/TaskListTest.java

```diff
@@ -0,0 +1,18 @@
+package seedu.duke;
+import seedu.duke.Task;
+import java.util.ArrayList;
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.fail;
+
+class TaskListTest {
+
+    @Test
+    public void test_loading() throws Exception {
```

Comment:
Perhaps you could make the test method name more descriptive? You could consider using a three part format, i.e. `featureUnderTest_testScenario_expectedBehavior()`